### ISSUE TITLE
Handling Amazon ECR

### ISSUE THREAD

**Issue by bastienbc** [flux-web/flux-web#20]
Hi,

I really like your project. It was exactly a missing part of my kubernetes CD stack.
But my society use an AWS ECR registry, and despite the fact that I give the credentials for your backend to scrape it, It shows no image and a default "latest" tag.

Could you add an ECR handler to your backend, maybe disabled by default, and enabled if some environment variable is set.

Thx!

![image](https://user-images.githubusercontent.com/25154212/67097319-25afcc80-f1ba-11e9-9f85-c1f481d4c17d.png)

**Comment by idobry** [comment]
Hey @bastienbc ,
flux-web's backend doesn't do that instead, he relies on flux's API *only*
can you please share your `values.yaml` file?

**Comment by bastienbc** [comment]
I don't have it now but I mainly took the helm release example in the Readme and changed the namespace to another one, and set the Flux service URL to point to the correct namespace.

If flux-web only relies on the Flux api, how can the fluxctl list-workloads be correct and the web not? I thought this was caused by something the backend would have done with the Flux results sorry.

But this means that this issue and the other I opened with the bad display on some images (#23) are the same and this one should be closed maybe...
